spec/shared_behaviors/taggable.rb

```diff
@@ -139,7 +149,12 @@ def tag_link(s)
         '#-initialhyphen'        => '-initialhyphen',
         '#-initialhyphen tag'    => '-initialhyphen',
         '#-initial-hyphen'       => '-initial-hyphen',
-        "\u202a#\u200eUSA\u202c" => 'USA'
+        "\u202a#\u200eUSA\u202c" => 'USA',
+        "test #ինչո՞ւ test"		 => "ինչո՞ւ",
+        "test #այո՜ո test"		 => "այո՜ո",
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.